Wrapping/Generators/Python/Tests/PythonTemplateTest.py

```diff
@@ -90,6 +90,15 @@
 except:
     pass
 
+# wwith a wrong file name
```

Comment:
wwith -> with

Reply:
Good catch. I had copy pasted the string that already contained the error. I updated my PR and corrected the original mistake too.